assets/pages/contributions-landing/newContributionsLanding.jsx

```diff
@@ -43,43 +33,147 @@ const defaultContributeCopy = `
   as and when you feel like it – choose the option that suits you best.
 `;
 
+type CountryMetaData = {
+  headerCopy: string,
+  contributeCopy: string,
+  currency: Object,
+  contribution: Object
+};
+
 const countryGroupSpecificDetails: {
```

Comment:
You should be able to get rid of this entire object using stuff in `assets/helpers/` - happy for that to be in a separate PR but I definitely think it should be done before putting the test live

Reply:
Agreed.